### DISCUSSION TITLE
Problem with GitHub Pages

### DISCUSSION THREAD

**Discussion by Nzii3** [retypeapp/retype#472] [Q&A] [answered]
I followed the steps for setting it up on GitHub pages and it looks like this, im using the `retype-action.yml` and deploying from the `retype` branch. I did do `retype build`.
![image](https://user-images.githubusercontent.com/106406461/210179507-82900838-548d-41ba-9174-8a8d4bdefc27.png)

**Comment by geoffreymcgill** [comment]
The GitHub Pages guide should be able to help, see 
https://retype.com/hosting/github-pages/

**Comment by Nzii3** [comment, marked as answer]
It was fixed, I just had to wait a little while then it rendered.
